FreeRTOS/Source/portable/ThirdParty/GCC/RISC-V/port.c

```diff
@@ -81,7 +81,7 @@
 
 #include "riscv_hal.h"
 
-#ifdef __riscv64
+#if __riscv_xlen == 64

```

Comment:
You use `STORE`/`LOAD` as names here but in portASM.S you used `store_x`/`load_x`. Since `STORE`/`LOAD` already existed, change `store_x`/`load_x` for consistency.